doc/source/mast_data_products/hap-parameters.rst

```diff
@@ -3,10 +3,46 @@ HAP Parameters
 
 .. _hap-parameters:
 
-Shown below are the parameters that are used by the Hubble Advanced Products. We include the parameter, the default value for WFC3 processing, and a description of that parameter.
+In the high-level svm_parameters or mvm_parameters directories, the instrument/detector-dependent configuration index files (e.g., acs_sbc_index.json) provide a map to the configuration files used for the major HAP processing steps. The svm_parameter configuration files are also used for pipeline products.
+
+
+For the astrodrizzle step, there are different files which should be employed depending upon the number of exposures int the visit.
+
+
+Detector Configuration Files
+****************************
+
+acs_hrc_index.json
+
+.. code-block:: python
+
+  {
+      "alignment": {
+          "all": "acs/hrc/acs_hrc_alignment_all.json"
+      },
+      "astrodrizzle": {
+          "any_n1": "acs/hrc/acs_hrc_astrodrizzle_any_n1.json",
+          "acs_hrc_any_n2": "acs/hrc/acs_hrc_astrodrizzle_any_n2.json",
+          "acs_hrc_any_n4": "acs/hrc/acs_hrc_astrodrizzle_any_n4.json",
+          "acs_hrc_any_n6": "acs/hrc/acs_hrc_astrodrizzle_any_n6.json",
+          "filter_basic": "any/any_astrodrizzle_filter_hap_basic.json",
+          "single_basic": "any/any_astrodrizzle_single_hap_basic.json",
+          "total_basic": "acs/hrc/acs_hrc_astrodrizzle_any_total.json"
+      },
+      "catalog generation": {
+          "all": "acs/hrc/acs_hrc_catalog_generation_all.json"
+      },
+      "quality control": {
+          "all": "acs/hrc/acs_hrc_quality_control_all.json"
+      }
+  }
+
+The different files for "astrodrizzle" are for the case of 1 exposure ("any_n1"), 2 exposures (acs_hrc_any_n2), etc. Each json file with an associated task above (e.g. astrodrizzle) will have the parameters for that task. These files will be used to initialize variables that are required in the processing of the data and, in the case of "astrodrizzle", are separated into different steps.
+
+Shown below are the parameters that are used by the Hubble Advanced Products. As an example, we include the parameter, the default value for WFC3/IR processing, and a description of that parameter.
 
 .. run_hap_processing
-.. identified in json files. 
+.. identified in json files.
 
```

Comment:
The discussion here seems out of context.  If you are describing the contents of a configuration file, provide the name of the specific file, and then the contents.  Note each numbered section maps to a corresponding section of the NameOfConfigurationFile.

Reply:
Sounds good. I've added the example filename to the docs.